tag/nitag.yml

```diff
@@ -0,0 +1,1814 @@
+      description: Get the value of a tag's "count" aggregate
+      operationId: GetTagCountAggregate
+      parameters:
+        - in: path
+          name: path
+          description: Tag path
+          type: string
+          required: true
+      responses:
+        200:
+          description: Count aggregate value
+          schema:
+            type: string
+        204:
+          description: No aggregate value found
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
+          description: Tag with the given path was not found
+        default:
+          $ref: '#/responses/Error'
+  /v2/tags/{path}/reset-aggregates:
+    post:
+      tags:
+        - tagAggregates
+      summary: 'Reset the aggregates of a tag'
+      description: Reset tag aggregates
+      operationId: ResetTagAggregates
+      parameters:
+        - in: path
+          name: path
+          description: Tag path
+          type: string
+          required: true
+      responses:
+        200:
+          description: Success
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
+          description: Tag with the given path was not found
+        default:
+          $ref: '#/responses/Error'
+  /v1/tags-rpc/update-current-values:
+    post:
+      tags:
+        - tags
+      summary: Update the current values of multiple tags
+      description: Push multiple current value updates to multiple tags in one request. Note that this route is deprecated.
+      operationId: RpcUpdateTagCurrentValues
+      parameters:
+        - in: body
+          name: updates
+          description: Updates
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/TagUpdate'
+      responses:
+        200:
+          description: Success
+        400:
+          description: Bad Request
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v2/update-current-values:
+    post:
+      tags:
+        - tags
+      summary: Update the current values of multiple tags (SystemLink Server only)
+      description: Push multiple current value updates to multiple tags in one request.
+      operationId: UpdateTagCurrentValues
+      parameters:
+        - in: body
+          name: updates
+          description: Updates
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/TagUpdate'
+      responses:
+        200:
+          description: Success
+        400:
+          description: Bad Request
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v2/tags/{path}/update-values:
+    post:
+      tags:
+        - tags
+      summary: Send multiple value updates to a tag
+      description: Push a list of current value update objects to update the value of the tag at the path multiple times. This can be used to send value history in bulk.
```

Comment:
Possible rewording, but I'm not sure if this is accurate:

Updates the current value of a tag at multiple times you specify. Use this route to send tag value history in bulk.

Reply:
That's accurate